movies-app/src/components/features/shows/ShowDetails/ShowDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Flex,
+  Heading,
+  Card,
+  CardHeader,
+  CardBody,
+  Stack,
+  StackDivider,
+  Box,
+  Text,
+  Image,
+} from "@chakra-ui/react";
+import { IShow } from "../../../../typings/Show.type";
+
+export interface IShowProps {
+  show: IShow;
+}
+
+export const ShowDetails = ({ show }: IShowProps) => {
+  return (
+    <Flex marginBottom="10">
```

Comment:
What are you using this Flex for? It contains only a single child element which is a bit weir for a Flex element.